src/keyboard/Keyboard.tsx

```diff
@@ -29,8 +29,7 @@ import { BehaviorBindingPicker } from "../behaviors/BehaviorBindingPicker";
 import { produce } from "immer";
 import { LockStateContext } from "../rpc/LockStateContext";
 import { LockState } from "@zmkfirmware/zmk-studio-ts-client/core";
-import { deserializeLayoutZoom, LayoutZoom } from "./PhysicalLayout";
-import { useLocalStorageState } from "../misc/useLocalStorageState";
```

Comment:
`useLocalStorageState` was created for future local storage uses. I don't think we should remove it https://github.com/zmkfirmware/zmk-studio/pull/31#discussion_r1771781964

Reply:
restored `useLocalStorageState`, i slightly modified the hook to have default serialize+deserializer, imho, these options should only be required for specific serialize+deserializer functions. lmk what you think ^^